include/execution.hpp

```diff
@@ -2186,6 +2437,12 @@ namespace std::execution {
         noexcept(nothrow_tag_invocable<split_t, _Sender>) {
         return tag_invoke(split_t{}, (_Sender&&) __sndr);
       }
+      template <sender _Sender>
+        requires (!__tag_invocable_with_completion_scheduler<split_t, set_value_t, _Sender>) &&
+          (!tag_invocable<split_t, _Sender>)
+      sender auto operator()(_Sender&& __sndr) const {
```

Comment:
I've been burned so many time by automatically deduced return types that I tend to avoid them in code that matters. When I know the type, I make it explicit.